Content.Server/GameObjects/Components/ActionBlocking/HandcuffComponent.cs

```diff
@@ -0,0 +1,249 @@
+﻿using Content.Server.GameObjects.EntitySystems.DoAfter;
+using Content.Shared.GameObjects.EntitySystems;
+using Content.Shared.Interfaces;
+using Content.Shared.Interfaces.GameObjects.Components;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using Content.Server.GameObjects.Components.GUI;
+using Robust.Shared.Serialization;
+using Robust.Shared.Log;
+using Robust.Shared.Localization;
+using Robust.Shared.ViewVariables;
+using Robust.Server.GameObjects.EntitySystems;
+using Content.Shared.GameObjects.Components.ActionBlocking;
+using Content.Server.GameObjects.Components.Mobs;
+using Robust.Shared.Maths;
+using System;
+
+namespace Content.Server.GameObjects.Components.ActionBlocking
+{
+    [RegisterComponent]
+    public class HandcuffComponent : SharedHandcuffComponent, IAfterInteract
+    {
+#pragma warning disable 649
```

Comment:
pragmas not needed anymore.

Reply:
That day was today https://github.com/space-wizards/space-station-14/pull/1893